src/webnn_native/ops/Clamp.h

```diff
@@ -15,29 +15,29 @@
 #ifndef WEBNN_NATIVE_OPS_CLAMP_H_
 #define WEBNN_NATIVE_OPS_CLAMP_H_
 
-#include "webnn_native/Graph.h"
-#include "webnn_native/Operand.h"
-#include "webnn_native/ops/Constant.h"
+#include "webnn_native/ops/ClampBase.h"
 
 namespace webnn_native { namespace op {
 
-    class Clamp final : public OperandBase {
+    class Clamp final : public ClampBase, public OperandBase {
       public:
-        Clamp(GraphBuilderBase* builder, OperandBase* input, ClampOptions const* options);
+        Clamp(GraphBuilderBase* builder, OperandBase* input, ClampOptions const* options)
+            : ClampBase(options), OperandBase(builder, {input}) {
```

Comment:
Clamp Operand don't need `ClampBase `now.

Reply:
Do we need the `ClampBase `?

Reply:
ClampOperator and Clamp share the codes from ClampBase for setting and getting options, any suggestions? @fujunwei